.github/fabricbot.json

```diff
@@ -1,3600 +0,0 @@
-                  "name": "removedFromProjectColumn",
-                  "parameters": {
-                    "projectName": "Technical 7.1"
-                  }
-                }
-              ]
-            }
-          ]
-        },
-        "eventType": "pull_request",
-        "eventNames": [
-          "pull_request",
-          "issues",
-          "project_card"
-        ],
-        "taskName": "Remove milestone if the PR is removed from the project column",
-        "actions": [
-          {
-            "name": "removeMilestone",
-            "parameters": {}
-          }
-        ]
-      },
-      "disabled": false
-    },
-    {
-      "taskType": "trigger",
-      "capabilityId": "IssueResponder",
-      "subCapability": "IssuesOnlyResponder",
-      "version": "1.0",
-      "id": "-NV3BbbFn",
-      "config": {
-        "conditions": {
-          "operator": "and",
-          "operands": [
-            {
-              "name": "isOpen",
-              "parameters": {}
-            },
-            {
-              "name": "labelAdded",
-              "parameters": {
-                "label": "In-PR :rocket:"
-              }
-            },
-            {
-              "name": "isInProjectColumn",
-              "parameters": {
-                "projectName": "Bugs 7.1",
-                "columnName": "To do"
-              }
-            }
-          ]
-        },
-        "eventType": "issue",
-        "eventNames": [
-          "issues",
-          "project_card"
-        ],
-        "actions": [
-          {
-            "name": "moveToProjectColumn",
-            "parameters": {
-              "projectName": "Bugs 7.1",
-              "columnName": "In progress"
-            }
-          },
-          {
-            "name": "addLabel",
-            "parameters": {
-              "label": "in progress :construction:"
-            }
-          }
-        ],
-        "taskName": "Move to In-progress column in the project once In-PR label applied"
-      }
-    },
-    {
-      "taskType": "trigger",
-      "capabilityId": "IssueResponder",
-      "subCapability": "IssuesOnlyResponder",
-      "version": "1.0",
-      "id": "-wiOMu4Ut",
-      "config": {
-        "conditions": {
-          "operator": "and",
-          "operands": [
-            {
-              "name": "isOpen",
-              "parameters": {}
-            },
-            {
-              "name": "labelAdded",
-              "parameters": {
-                "label": "In-PR :rocket:"
-              }
-            },
-            {
-              "name": "isInProjectColumn",
-              "parameters": {
-                "projectName": "Features 7.1",
-                "columnName": "To do"
-              }
-            }
-          ]
-        },
-        "eventType": "issue",
-        "eventNames": [
-          "issues",
-          "project_card"
-        ],
-        "taskName": "Move to In-progress column in the project once In-PR label applied",
```

Comment:
Project board automation is not supported in the new service, so related tasks were not ported.